File: python_draft/src/cpp/type2/process_data_original.cpp
#include <vector>
#include <algorithm>
std::vector<int> processData(const std::vector<int>& input){
    std::vector<int> result;
    for(int item : input){
        if(item % 2 == 0) result.push_back(item * 2);
    }
    return result;
}

File: python_draft/src/java/type2/ProcessDataOriginal.java
public class ProcessDataOriginal {
public static java.util.List<Integer> processData(java.util.List<Integer> input) {
    java.util.List<Integer> result = new java.util.ArrayList<>();
    for (int item : input) {
        if (item % 2 == 0) {
            result.add(item * 2);
        }
    }
    return result;
}

}
